.swiftlint.yml

```diff
@@ -0,0 +1,5 @@
+disabled_rules:
+  - function_body_length
+  - line_length
```

Comment:
I'm thinking I might try to re-enable `line_length` and experiment with some long line wrapping styles.